frontend/src/scenes/cohorts/CohortFilters/types.ts

```diff
@@ -86,6 +88,7 @@ export interface Row {
 
 export interface CohortFieldBaseProps extends Omit<CohortFieldLogicProps, 'cohortFilterLogicKey'> {
     cohortFilterLogicKey?: string
+    cohortId: CohortLogicProps['id']
```

Comment:
why do we need the cohortId prop in the logic, doesn't seem to be used? 🤔

Reply:
cohortId prop is used by useIsReadonlyCohort hook in different cohort fields.
![image](https://github.com/PostHog/posthog/assets/79001785/c2b45bf8-8e05-4c25-9b6a-6cfac502b496)

Reply:
I see, that will go away then when you remove the hook 👍

Reply:
this code is a bit old / in need of polishing, but basically you can use `BindLogic` so you don't have to pass props around for `cohortEditLogic`, and this should be at the top level - search for examples in the codebase for an idea of how this works, and see https://keajs.org/docs/react/bindlogic/